tests/util/permutation_test.py

```diff
@@ -0,0 +1,30 @@
+# Copyright (c) Facebook, Inc. and its affiliates.
+#
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+import numpy as np
+
+import compiler_gym.util.permutation as permutation
+from tests.test_main import main
+
+
+def test_permutation_number_mapping():
+    original_permutation = np.array([4, 3, 1, 5, 2, 6, 0], dtype=int)
+    permutation_number = permutation.convert_permutation_to_number(original_permutation)
+    mapped_permutation = permutation.convert_number_to_permutation(
+        n=permutation_number, permutation_size=len(original_permutation)
+    )
+    assert np.array_equal(original_permutation, mapped_permutation)
```

Comment:
Nitpick: please use `np.testing.assert_array_equal(...)` instead of `assert np.array_equal(...)` as it produces better error messages on failure.

Reply:
Done.